src/views/components-default/TheDefaultHeader.vue

```diff
@@ -36,34 +36,16 @@
             Buy ETH
           </a>
         </v-col>
-        <v-col
-          cols="4"
-          :class="$vuetify.breakpoint.mdAndUp ? 'text-center' : 'text-right'"
-        >
-          <mew-button
-            class="px-2"
-            :title="$vuetify.breakpoint.mdAndUp ? 'MEW tools' : ''"
-            color-theme="white"
-            :has-full-width="false"
-            btn-size="large"
-            btn-style="outline"
-            :icon="
-              $vuetify.breakpoint.mdAndUp
-                ? 'mdi-view-module'
-                : require('@/assets/images/icons/icon-grid-dot.png')
-            "
-            :icon-type="$vuetify.breakpoint.mdAndUp ? 'mdi' : 'img'"
-            icon-align="left"
-            @click.native="mewTools = true"
-          />
+        <v-col cols="4" class="text-right">
+          <mew-tools class="ml-auto" />
         </v-col>
       </v-row>
     </v-container>
   </div>
 </template>
 
 <script>
-import mewTools from '@/components/mewTools/MewTools';
+import mewTools from '@/components/mew-tools/MewTools';
```

Comment:
When you import ui components please keep the same name casing so its 
MewTools

Reply:
Got it!!